homework/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="main.css">
+  <title>Git content fetching </title>
+</head>
+
+<body>
+
+  <div class="container">
+    <div class="topDiv">
+      <label for="">HYF Repositories </label>
```

Comment:
Either declare which control the label is for (using its `id`), or just leave the `for` all together.

Reply:
i remove the for!